### DISCUSSION TITLE
Is milvus-sdk-rust suitable to use?

### DISCUSSION THREAD

**Discussion by pycui** [milvus-io/milvus#36678] [Q&A and General discussion]
It seems https://github.com/milvus-io/milvus-sdk-rust hasn't been updated for a while. If feature completeness is not a concern, is the library still suitable to be used? Thanks

**Comment by yhmo** [comment]
Very few users use rust sdk, there might be unknown bugs in the sdk. Personally, I don't recommend using this sdk. But if you are a strong rust developer, and willing to improve the sdk, we will appreciate it.

**Comment by xiaofan-luan** [comment]
> Suggest an answer

Don't think it's ready yet. We plan to release it with milvus 3.0.
Right now cpp, go, java, python and nodejs is what we've been mainly working on
